bdb_transaction_cli/cli.py

```diff
@@ -1,15 +1,132 @@
-# -*- coding: utf-8 -*-
-
 import click
+import json
+
+from bigchaindb_common.crypto import generate_key_pair
+from bigchaindb_common.transaction import Transaction, \
+    Condition, Fulfillment, Metadata, Asset, Ed25519Fulfillment
+
+from bdb_transaction_cli.utils import json_argument, json_option, listify
+
+
+@click.group()
```

Comment:
As far as I remember @TimDaub told me it was a workaround on some `click` limitation, but I don't remember exactly why.